http4s/client/src/test/scala/endpoints/http4s/client/Http4sClientEndpointsJsonSchemaTest.scala

```diff
@@ -0,0 +1,58 @@
+package endpoints.http4s.client
+
+import endpoints.algebra
+import endpoints.algebra.client
+
+import cats.effect.Sync
+import org.http4s.client.Client
+import cats.effect.IO
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.global
+import org.http4s.client.blaze.BlazeClientBuilder
+import cats.effect.ContextShift
+import endpoints.algebra.circe
+import org.http4s.Uri
+
+class TestJsonSchemaClient[F[_]: Sync](host: Uri, client: Client[F])
+    extends Endpoints[F](host, client)
+    with BasicAuthentication
+    with JsonEntitiesFromCodecs
+    with algebra.BasicAuthenticationTestApi
+    with algebra.EndpointsTestApi
+    with algebra.JsonFromCodecTestApi
+    with circe.JsonFromCirceCodecTestApi
+    with circe.JsonEntitiesFromCodecs {}
+
+class Http4sClientEndpointsJsonSchemaTest
+    extends client.EndpointsTestSuite[TestJsonSchemaClient[IO]]
+    with client.BasicAuthTestSuite[TestJsonSchemaClient[IO]]
+    with client.JsonFromCodecTestSuite[TestJsonSchemaClient[IO]] {
```

Comment:
It would be nice to add `client.ChunkedJsonEntitiesTestSuite` as well. You will have to make `TestJsonSchemaClient` extend `ChunkedJsonEntities` and `algebra.ChunkedJsonEntitiesTestApi`

Reply:
Done